### ISSUE TITLE
[BUG] Stream Deck Studio disconnects from Companion

### ISSUE THREAD

**Issue by WaffleTime16** [bitfocus/companion#3175]
### Is this a bug in companion itself or a module?

- [x] I believe this to be a bug in companion

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Describe the bug

Using an SDS on PoE in the same subnet as the Companion server.  After 10-30 minutes or so, the SDS will revert to the Welcome display and become unresponsive until it is power cycled.  Buttons also will not detect it in this state.  Restarting Companion does not fix it.  In the Companion log, it shows the Discovery process for the SDS and connects.  When it dies the log registers it as it losing the TCP connection.  

Using Companion 3.4.3 stable and ubuntu 24.04 server.  

### Steps To Reproduce

1. Connect the SDS to the same network as Companion server
2. Register the surface
3. The SDS will eventually disconnect

### Expected Behavior

The SDS will remain connected to the Companion instance.  

### Environment (please complete the following information)

```markdown
- OS: Ubuntu Server 24.04
- Browser: Chrome
- Companion Version: 3.4.3
```

**Comment by Julusian** [comment]
What firmware is your SDS running? It gives the version number on the screen along with the ip address when companion is not connected.

I'm not able to reproduce this myself after leaving it for an hour, I think I am running firmware `1.05.007` (I don't want to disconnect to check)

**Comment by WaffleTime16** [comment]
Just checked and it's running 1.05.009

**Comment by Julusian** [comment]
48 hours in, and no crash (as far as I can tell).

I have another couple of ideas that it would be good if you could try:

1) Does it still crash even when nothing connects to it? (remove the config in companion/buttons telling them to connect to it, and see what happens)

2) Does it get better if you put it on its own physical network or vlan with just the SDS and companion?

The theory for both of these is that maybe it isn't companion at fault, but some other network traffic that is causing the crash. There is this warning in the buttons documentation:
![Image](https://github.com/user-attachments/assets/f4ba302d-2381-412c-bd84-51ed6b2c6d99)

**Comment by WaffleTime16** [comment]
OK, that's an interesting and viable theory.  We do have a bit of Dante traffic on the same network at the moment, about 4 hosts that are sending maybe a total of 6 channels back and forth at the moment.  

Had the SDS running disconnected on its own for 5 minutes now, most of the time it would've already locked up by this point.  

I did reach out on Slack and they directed me here to post a bug report.  It's a bit confusing to figure out where to communicate issues with this device.  I'll follow up in the #Buttons slack as well.

**Comment by Julusian** [comment]
> I did reach out on Slack and they directed me here to post a bug report. It's a bit confusing to figure out where to communicate issues with this device. I'll follow up in the #Buttons slack as well.

Yeah its a situation of companion using a different implementation of talking to the SDS, so it could be something wrong there. So I guess they wanted us to consider that possibility.

But as mine appears to be fine, that suggests that there are some conditions I am not triggering..

Interestingly I do have dante on my network too. An x32, a couple of virtual soundcard and a handful of avio, so by itself that might not be the issue. But I had the sds a switch away from that, and my dante is probably entirely unicast traffic with igmp snooping on the switch so nothing extra should be reaching the sds

**Comment by WaffleTime16** [comment]
That's about the same as what we have.  I don't have access to configuration for the switch we are using in the building, so it's hard to say if there is some sort of traffic blocking going on that's triggering it.  I sandboxed the SDS on an isolated network with a few Dante devices and it seemed fine.  So I think it's a reasonable assumption that it's some traffic or device on our production network that is causing us the issue.  I followed up on the Buttons slack as well.

**Comment by WaffleTime16** [comment]
I've recently heard from Elgato and thru the Buttons slack that this is a firmware issue that is being worked on.

**Comment by Julusian** [comment]
Hopefully this is fixed in the latest firmware of the SDS. Companion should prompt you to upgrade

**Comment by Julusian** [comment]
@WilcoVanMinderhout The 'Firmware Utility' is the recommended way to update the SDS firmware https://user.bitfocus.io/download
